.github/workflows/ci.yml

```diff
@@ -15,14 +15,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ['3.7', '3.8', '3.9', '3.10']
-        django: ['3.2', '4.0']
-        mozilla_django_oidc: ['1.2', '2.0']
+        python: ['3.8', '3.9', '3.10', '3.11']
+        django: ['3.2', '4.1', '4.2']
+        mozilla_django_oidc: ['2.0', '3.0']
```

Comment:
self-reminder: dropping support for mozilla-django-oidc 1.x will require a major version bump in this library